src/css/custom.css

```diff
@@ -39,3 +44,16 @@
   background: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='white' d='M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12'/%3E%3C/svg%3E")
     no-repeat;
 }
+
+.header-slack-link:hover {
+  opacity: 0.6;
+}
+
+.header-slack-link::before {
+  content: '';
+  width: 24px;
+  height: 24px;
+  display: flex;
+  background: url("data:image/svg+xml;charset=utf-8,%3Csvg%20viewBox%3D%220%200%2060%2060%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Cpath%20d%3D%22m21.902.148c-3.299%200-5.973%202.68-5.973%205.985a5.979%205.979%200%200%200%205.973%205.985h5.974v-5.985a5.98%205.98%200%200%200%20-5.974-5.985m0%2015.96h-15.929c-3.299%200-5.973%202.68-5.973%205.986%200%203.305%202.674%205.985%205.973%205.985h15.93c3.298%200%205.973-2.68%205.973-5.985%200-3.306-2.675-5.986-5.974-5.986%22%20fill%3D%22%23097eff%22%3E%3C%2Fpath%3E%3Cpath%20d%3D%22m59.734%2022.094c0-3.306-2.675-5.986-5.974-5.986s-5.973%202.68-5.973%205.986v5.985h5.973a5.98%205.98%200%200%200%205.974-5.985m-15.929%200v-15.961a5.98%205.98%200%200%200%20-5.974-5.985c-3.299%200-5.973%202.68-5.973%205.985v15.96c0%203.307%202.674%205.987%205.973%205.987a5.98%205.98%200%200%200%205.974-5.985%22%20fill%3D%22%23097eff%22%3E%3C%2Fpath%3E%3Cpath%20d%3D%22m37.831%2060a5.98%205.98%200%200%200%205.974-5.985%205.98%205.98%200%200%200%20-5.974-5.985h-5.973v5.985c0%203.305%202.674%205.985%205.973%205.985m0-15.96h15.93c3.298%200%205.973-2.68%205.973-5.986a5.98%205.98%200%200%200%20-5.974-5.985h-15.929c-3.299%200-5.973%202.68-5.973%205.985a5.979%205.979%200%200%200%205.973%205.985%22%20fill%3D%22%23ecb12f%22%3E%3C%2Fpath%3E%3Cpath%20d%3D%22m0%2038.054a5.979%205.979%200%200%200%205.973%205.985%205.98%205.98%200%200%200%205.974-5.985v-5.985h-5.974c-3.299%200-5.973%202.68-5.973%205.985m15.929%200v15.96c0%203.306%202.674%205.986%205.973%205.986a5.98%205.98%200%200%200%205.974-5.985v-15.961a5.979%205.979%200%200%200%20-5.974-5.985c-3.299%200-5.973%202.68-5.973%205.985%22%20fill%3D%22%23ecb12f%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E")
```

Comment:
Is there a specific source for this background image?

Reply:
I found the corresponding SVG code in the SLACK official website and generated the corresponding inline format.